test/unit/firestore.js

```diff
@@ -125,10 +125,12 @@ describe('MockFirestore', function () {
             transaction.update(db.doc('doc'), {
               name: 'abc'
             });
+            return 'cba';
           });
-        }).then(function() {
+        }).then(function(transactionReturn) {
           db.doc('doc').get().then(function(doc2) {
             expect(doc2.get('name')).to.equal('abc');
+            expect(transactionReturn).to.equal('cba');
```

Comment:
please put this verification into a separate unit test `it('transaction retruns expected value')`